src/frontend/Komponenter/Personoversikt/Behandlingsoversikt.tsx

```diff
@@ -158,14 +163,16 @@ const Behandlingsoversikt: React.FC<{ fagsakId: string }> = ({ fagsakId }) => {
                     />
                     {kanStarteRevurdering && (
                         <>
-                            <KnappMedMargin onClick={() => settVisRevurderingvalg(true)}>
-                                Opprett ny behandling
-                            </KnappMedMargin>
-                            <RevurderingModal
-                                visModal={visRevurderingvalg}
-                                settVisModal={settVisRevurderingvalg}
+                            <LagBehandlingModal
+                                visModal={visLagBehandlingModal}
+                                settVisModal={settVisLagBehandlingModal}
                                 fagsakId={fagsakId}
+                                settHentTilbakekrevinger={settHentTilbakekrevingsbehandlinger}
```

Comment:
Litt usikker på denne - lage ny useEffekt og bruke Dispatch - > hentTilbakekrevinger?

Reply:
Alternativ approach:
- send med en prop som heter `relastTilbakekrevingBehandlinger={() => {hentTilbakekrevingBehandlinger()}}` ? Må jo ikke bruke useEffect-hooks for å utføre en relast?